lib/msal-browser/src/config/FeatureFlags.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// feature flag configuration for new features in MSAL.js
+export type FeatureSupportConfiguration = {
+    enablePlatformBrokerDOMSupport?: boolean;
```

Comment:
Do we really need a separate config to control DOM vs extension support?

Reply:
I suggested it to gate this feature and an easy way to turn it off if required.

Reply:
Maybe I'm missing something, why isn't the existing `allowPlatformBroker` flag sufficient for turning off the feature?